src/pages/Authorization.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Box, FormControlLabel, Checkbox } from "@mui/material";
+import LoginForm from "components/LogInForm";
+import RegistrationForm from "components/RegistrationForm";
+
+const Authorization: React.FC = () => {
+  const [isNew, setIsNew] = React.useState(false);
+
+  const handleChange = (event: {
+    target: { checked: boolean | ((prevState: boolean) => boolean) };
+  }) => {
+    setIsNew(event.target.checked);
+  };
+
+  return (
+    <Box
+      sx={{ border: 1, borderColor: "grey.500" }}
+      width="400px"
+      bgcolor="#F0F8FF"
+      marginTop="30px"
+      height="auto"
+      padding="20px"
+    >
+      {isNew ? <RegistrationForm /> : <LoginForm />}
+      <FormControlLabel
```

Comment:
It should be one component with form (e.g. `AuthForm`). Checkbox should be inside of it. Use similar conditional rendering in the form to change its texts and inputs.